hubble/hubble.go

```diff
@@ -161,6 +163,18 @@ func NewK8sHubble(client k8sHubbleImplementation, p Parameters) *K8sHubble {
 }
 
 func (k *K8sHubble) Log(format string, a ...interface{}) {
+	if k.params.RedactHelmCertKeys {
+		formattedString := fmt.Sprintf(format+"\n", a...)
+		for _, certKey := range []string{
+			certs.EncodeCertBytes(k.certManager.CAKeyBytes()),
+		} {
+			if certKey != "" {
+				format = strings.ReplaceAll(formattedString, certKey, "[--- REDACTED WHEN PRINTING TO TERMINAL (USE --redact-helm-certificate-keys=false TO PRINT) ---]")
+			}
+		}
+		fmt.Fprint(k.params.Writer, format)
```

Comment:
```suggestion
			if certKey != "" {
				formattedString = strings.ReplaceAll(formattedString, certKey, "[--- REDACTED WHEN PRINTING TO TERMINAL (USE --redact-helm-certificate-keys=false TO PRINT) ---]")
			}
		}
		fmt.Fprint(k.params.Writer, formattedString)
```

otherwise the format specifiers won't be interpreted, leading to e.g. the following output on `cilium uninstall`:
 
```
🔥 Deleting Hubble UI...🔥 Deleting Hubble UI certificates...🔥 Deleting Relay...🔥 Deleting Relay certificates...✨ Patching ConfigMap %s to disable Hubble...🚀 Creating ConfigMap for Cilium version %s...♻️  Restarted Cilium podsℹ️  Storing helm values file in %s/%s Secret✅ Hubble was successfully disabled.🔮 Auto-detected Kubernetes kind: GKE
```